clients/admin-ui/src/pages/management/consent.tsx

```diff
@@ -0,0 +1,508 @@
+              >
+                TCF status:{" "}
+                {isTcfEnabled ? (
+                  <Badge backgroundColor="green.100">Enabled </Badge>
+                ) : (
+                  <Badge backgroundColor="red.100">Disabled</Badge>
+                )}
+              </Text>
+              <Text
+                fontSize="sm"
+                lineHeight="5"
+                fontWeight="medium"
+                color="gray.700"
+              >
+                To {isTcfEnabled ? "disable" : "enable"} TCF, please contact
+                your Fides Administrator or{" "}
+                <DocsLink href="https://ethyca.com">Ethyca support</DocsLink>
+              </Text>
+            </Box>
+
+            <Box
+              mt="24px"
+              backgroundColor="gray.50"
+              borderRadius="4px"
+              padding="14px"
+            >
+              <Text
+                fontSize="md"
+                fontWeight="bold"
+                lineHeight={5}
+                color="gray.700"
+                mb={3}
+              >
+                Vendor overrides
+              </Text>
+              {isTcfEnabled ? (
+                <>
+                  <Text
+                    marginBottom={2}
+                    fontSize="sm"
+                    lineHeight="5"
+                    fontWeight="medium"
+                    color="gray.700"
+                  >
+                    Override vendor purposes:{" "}
+                    <Switch
+                      size="sm"
+                      colorScheme="purple"
+                      isChecked={isOverrideEnabled}
+                      onChange={handleOverrideOnChange}
+                      isDisabled={isPatchConfigSettingsLoading}
+                    />
+                  </Text>
+                  <Text
+                    mb={2}
+                    fontSize="sm"
+                    lineHeight="5"
+                    fontWeight="medium"
+                    color="gray.700"
+                  >
+                    {isOverrideEnabled
+                      ? "The table below allows you to adjust which TCF purposes you allow as part of your user facing notices and business activites."
+                      : "Toggle on if you want to globally change any flexiable legal bases or remove TCF purposes from your CMP."}
+                  </Text>
+                </>
+              ) : null}
+              {isOverrideEnabled && isTcfEnabled ? (
+                <Text
+                  fontSize="sm"
+                  lineHeight="5"
+                  fontWeight="medium"
+                  color="gray.700"
+                >
+                  To configure this section, select the purposes you allow and
+                  where available, the appropriate legal basis (either Consent
+                  or Legitmate Intererest). Read the guide on{" "}
+                  <DocsLink href="https://ethyca.com">
+                    TCF Override here.{" "}
```

Comment:
Does the URL for this exist, or is this a placeholder until it does? Also, the designs have the entire `Read the guide on TCF Override here.` text as the link